package.json

```diff
@@ -1,57 +1,60 @@
 {
   "name": "react-hook-videojs",
-  "version": "2.1.0-beta.1",
+  "version": "2.1.0-beta.3",
   "description": "A simple react hook to easily integrate video.js with React",
   "author": "jimmycallin",
   "license": "MIT",
   "repository": "jimmycallin/react-hook-videojs",
+  "types": "./dist/types.d.ts",
   "main": "dist/react-hook-videojs.umd.js",
   "module": "dist/react-hook-videojs.es.js",
-  "source": "src/index.jsx",
+  "source": "src/index.tsx",
   "engines": {
     "node": ">=12"
   },
   "scripts": {
-    "clean": "rimraf dist",
-    "build": "vite build",
+    "clean": "rimraf dist .tmp",
+    "build": "vite build && tsc --project tsconfig.types.json",
     "test": "npm run test:lint && npm run test:jest && npm run test:formatting",
     "test:jest": "jest",
-    "test:lint": "eslint --ext .js --ext .jsx .",
+    "test:lint": "eslint ./src",
     "test:formatting": "prettier -c .",
+    "format": "prettier --write .",
     "prepare": "npm run clean && npm run build",
     "prepublishOnly": "npm run test"
   },
   "peerDependencies": {
-    "react": "^16.8.0, ^17.0.0",
+    "react": ">= 16.8.0 < 19",
```

Comment:
please remove updates to the code that are related to react 18, as we'll deal with that in the other PR